cpp/src/distance/specializations/detail/canberra_double_double_double_int.cu

```diff
@@ -1,33 +0,0 @@
-/*
- * Copyright (c) 2021-2023, NVIDIA CORPORATION.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
```

Comment:
Users downstream are still including these files in their code. Instead of removing all of these files, let's just remove their contents and add a deprecation warning that the files will be removed in 23.08 (or 23.10). The improvements you are making are already pretty massive, but I want to eliminate removing any files that are going to require synchronously updating all of the users before they get merged. In addition, we now have users outside of RAPIDS and our of courtesy we should be giving them time to update their code.